api/afdb/library.ts

```diff
@@ -1,8 +1,10 @@
+"use server"
```

Comment:
Is this needed? `server` is default, right?

Reply:
yes but I'm using server actions here